db/dump.py

```diff
@@ -40,6 +40,7 @@
         "build_sha1",
         "lossless",
         "submitted",
+        "gid_type"
```

Comment:
Can add a trailing comma here too. http://dave.cheney.net/2014/10/04/that-trailing-comma